.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+-   repo: https://github.com/pycqa/flake8
+    rev: '4.0.1'
```

Comment:
I'd prefer to keep this unpinned. Whenever pyflakes is updated it may create new failures, but those are also things we'd want to know about. Also we are going to want to add 3.10 testing pretty soon, and I don't know if this versions supports it yet.